frontend/src/components/TabbedResults.less

```diff
@@ -0,0 +1,40 @@
+.close {
+  width: auto;
```

Comment:
tab indent should be with [4 spaces](https://github.com/carlosms/gitbase-playground/blob/b55a986dd66ad8e642f5ccd90a21d277b4c7386f/.editorconfig#L13)...
We should open an issue to address this inconsistency...

Reply:
I created a new issue: https://github.com/src-d/gitbase-playground/issues/37